webpack.config.js

```diff
@@ -29,7 +29,7 @@
  * ---------------------------------------------------------------------
  */
 
-const CleanWebpackPlugin = require('clean-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
```

Comment:
could you explain this syntax ?

Reply:
appart that, ok for me

Reply:
This is related to the way exports as done on this new version. I do not know why it has been changed, but old way do not work.
Here is the migration guide for this lib https://github.com/johnagan/clean-webpack-plugin/issues/106